Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.yml

```diff
@@ -946,16 +940,18 @@ script:
     - name: toDate
       required: true
       description: To Date ( yyyy-MM-dd )
+    - name: entityTypeId
+      required: true
```

Comment:
Does it have to be required?
Because it breaks backward compatibility.

Reply:
removed it.